### PR TITLE
WIP feat(sledgehammer): CentOS 8 based Sledgehammer image.

### PR DESCRIPTION
**Pull request by VictorLowther**
This implements code for building a CentOS 8 based Sledgehammer with support for customizations during the build process.  You can customize what packages are added, manage which services enabled/disabled by default, and inject extra tasks during the build process to accommodate whatever specific tweaks you need.

This update to the builder code also supports the upcoming bootenv-customize changes coming in 4.4.x by creating a profile you can use to override the discovery and sledgehammer bootenvs to use the Sledgehammer image this content pack generates.

### CODE REVIEW

File: sledgehammer-builder/bootenvs/build-sledgehammer.yaml [hunk 1]
```diff
@@ -156,12 +154,10 @@ Templates:
       selinux-policy-minimum
       setup
       shadow-utils
+      # sipcalc
       squashfs-tools
-      stress
```
Comment (galthaus) [inline sledgehammer-builder/bootenvs/build-sledgehammer.yaml:160]: Did these go away or move somewhere else?  burnin needs them.

Comment (VictorLowther) [reply sledgehammer-builder/bootenvs/build-sledgehammer.yaml:160]: I know -- they are not available in the base repos anymore, and I have not hunted down an alternate source yet.

Probably have to pull them from Fedora like we do for sipcalc

Comment (galthaus) [reply sledgehammer-builder/bootenvs/build-sledgehammer.yaml:160]: Ah - okay - thanks.

Comment (VictorLowther) [reply sledgehammer-builder/bootenvs/build-sledgehammer.yaml:160]: OK, current code now pulls in stress from epel, and stress-ng from fedora 29 (which is what RHEL8 was based on)

**Pull request merged**